OPUNetGameSelectWnd.cpp

```diff
@@ -860,14 +838,7 @@ void OPUNetGameSelectWnd::OnClickJoin()
 
 void OPUNetGameSelectWnd::OnClickCreate()
 {
-	// Host
-	int errorCode;
-	MultiplayerPreGameSetupWnd multiplayerPreGameSetupWnd;
-	HostGameParameters hostGameParameters;
 	char password[16];

```

Comment:
There is a private class variable also named password. Recommend renaming. I think this could be named hostPassword. The class level variable could maybe be joinPasswordAttempt?

Reply:
That is a potential point of confusion. In either case, the password comes from the same text box. The difference is between hosting a game versus joining a game, where the password needs to be stored longer for one of them. I'll have to take another look at the code.